komoju-php/komoju-php/lib/komoju/KomojuApi.php

```diff
@@ -89,6 +94,13 @@ private function get($uri, $asArray = false)
     // );
     private function post($uri, $payload)
     {
+        $payload['fraud_details'] = array(
+            'customer_ip' => $_SERVER['REMOTE_ADDR'],
+            'customer_email' => $payload['customer_email'] ?? '',
+            'browser_language' => $_SERVER['HTTP_ACCEPT_LANGUAGE'],
+            'browser_user_agent' => $_SERVER['HTTP_USER_AGENT'],
+        );
```

Comment:
It just occurred to me that we don't actually need these fields for every POST request. It's only necessary when creating payments (so /api/v1/payments or /api/v1/sessions/{id}/pay). That said, our API ignores unexpected inputs and so this implementation is no problem.